src/core/client/admin/components/Comment/InReplyTo.tsx

```diff
@@ -12,9 +12,15 @@ interface Props {
 
 const InReplyTo: FunctionComponent<Props> = ({ children, onUsernameClick }) => {
   const Username = () => (
-    <BaseButton onClick={onUsernameClick} className={styles.usernameButton}>
-      <span className={styles.username}>{children}</span>
-    </BaseButton>
+    <Localized
+      id="common-username"
+      attrs={{ "aria-label": true }}
+      vars={{ username: children }}
+    >
+      <BaseButton onClick={onUsernameClick} className={styles.usernameButton}>
+        <span className={styles.username}>{children}</span>
+      </BaseButton>
+    </Localized>
```

Comment:
I think we have to do similar in the stream side `InReplyTo.tsx`. Check `src/core/client/stream/tabs/Comments/Comment/InReplyTo.tsx`.

Reply:
I didn't update that one because it's not an interactive element, and from what I've read non interactive elements should not have aria labels, and should have descriptive visible text that the screen reader can read, and my screen reader reads this as "in reply to ...".